ios/SalesforceReact/SFNetReactBridge.m

```diff
@@ -105,10 +105,11 @@ @implementation SFNetReactBridge
     [restApiInstance sendRequest:request
                                       failureBlock:^(id response, NSError *e, NSURLResponse *rawResponse) {
                                           __strong typeof(self) strongSelf = weakSelf;
+                                          NSMutableDictionary *responseDictionary = [[rawResponse asDictionary] mutableCopy];
+                                          responseDictionary[@"body"] = [strongSelf serializableResponse:response rawResponse:rawResponse];
                                           NSMutableDictionary *errorDictionary = [NSMutableDictionary new];
                                           errorDictionary[@"error"] = e.localizedDescription;
-                                          errorDictionary[@"urlResponse"] = [rawResponse asDictionary];
-                                          errorDictionary[@"response"] = [strongSelf serializableResponse:response rawResponse:rawResponse];
+                                          errorDictionary[@"response"] = responseDictionary;
                                           callback(@[RCTMakeError(@"sendRequest failed", nil, errorDictionary)]);
```

Comment:
Android returns a string currently while RCTMakeError gives a dictionary, should we change one of them?

Reply:
Let's use the same approach as what we did for hybrid?

Reply:
Maybe we should leave this as-is for now. We're very close to releasing `8.2`. Maybe we can line them up in `8.3`?

Reply:
On Android, we always send back strings (for success and error callbacks). The js side takes care of parsing the string into an object. See https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative/blob/dev/src/react.force.common.js#L63

Reply:
Ahh, thanks!